### PR TITLE
browser security settings

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Customer request for browser security details.

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-125208](https://oktainc.atlassian.net/browse/OKTA-125208)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
@robertdamphousse-okta or @lboyette-okta or @jmelberg-okta or @rchild-okta

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
This needs to get into 2017.21 ASAP.

File: _source/_code/javascript/okta_sign-in_widget.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The Okta Sign-In Widget is a JavaScript widget from Okta that gives
 you a fully featured and customizable login experience which
 can be used to authenticate users on any web site.
 
-{% img okta-signin.png alt:"Screenshot of basic Okta Sign-In Widget" %}
+![Screenshot of basic Okta Sign-In Widget](/assets/img/okta-signin.png)
```
Comment (jmelberg-okta) [inline _source/_code/javascript/okta_sign-in_widget.md]: Why did these change to the old format?

File: _source/_code/javascript/okta_auth_sdk.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ The Okta Auth SDK builds on top of our [Authentication API](/docs/api/resources/
 
 - An Okta org - Register for [Okta Developer Edition](https://www.okta.com/developer/signup/) if you don't have an existing org
 - A domain that is [CORS enabled for your org](/docs/api/getting_started/enabling_cors.html)
+- Okta requires these browser security settings:
+    * Enable active scripting.
```
Comment (jmelberg-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: @rchild-okta, @lboyette-okta Is there a better way to phrase this?

File: _source/_code/javascript/okta_sign-in_widget.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/_code/javascript/okta_sign-in_widget.md]: All the images are the old style, I'll fix these (intellij issue, my mistake).

File: _source/_code/javascript/okta_auth_sdk.md (hunk 2 above)
Comment (robertjd) [inline _source/_code/javascript/okta_auth_sdk.md]: Talked with Len, we should remove this note about active script

File: _source/_code/javascript/okta_auth_sdk.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ The Okta Auth SDK builds on top of our [Authentication API](/docs/api/resources/
...
+    * Enable third-party cookies.
+    * Add the Okta URL to the browser's list of trusted sites.
```
Comment (lboyette-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: This usually isn't necessary. This whole section can be summed up in:
`- Okta requires that browsers have third-party cookies enabled for your Okta org's domain.`

File: _source/_code/javascript/okta_auth_sdk.md (hunk 2 above)
Comment (lboyette-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: If anything, we should say that JavaScript should be enabled (I think that's implied, given it's a js library)

File: _source/_code/javascript/okta_auth_sdk.md (hunk 3 above)
Comment (rchild-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: And even in this case, isn't it only required in the iframe flow I.e. don't really care about it if we do the redirect code flow right?

Comment (rchild-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Yes - in fact, we explicitly do **not** want them to add it to the list of trusted sites - we've had issues with other flows in the past where we've told them to do this and it subtly messes with other things (i.e. there is a subset of orgs that use compatibility mode on trusted sites, which are normally reserved for internal apps).

Comment (mystiberry-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Great! @rchild-okta or @lboyette-okta just confirming your comments apply to sign-in widget as well as auth SDK. Will have a revision up shortly.

**Comment by mystiberry-okta** [comment]
Need review from @lboyette-okta @rchild-okta or @jmelberg-okta .

File: _source/_code/javascript/okta_sign-in_widget.md [hunk 4]
```diff
@@ -64,6 +64,10 @@ might occur in a user login flow.
 
 {% img okta-signin-validation-failure.png alt:"Screenshot of Okta Sign-In Widget displaying validation error" %}
 
+### Browser Security Settings
+
+Okta requires that browsers have third-party cookies enabled for your Okta org's domain.
```
Comment (jmelberg-okta) [inline _source/_code/javascript/okta_sign-in_widget.md]: nit: `org` -> `organization` . To keep with consistency throughout this file

File: _source/_code/javascript/okta_sign-in_widget.md [hunk 5]
```diff
@@ -540,5 +536,10 @@ var oktaSignIn = new OktaSignIn({
               { text: 'Rocket Sled Questions', href: 'http://acme.example.com/questions/rocket-sled' }
             ]
 
+## Troubleshooting
+
+If you are working with a single-page app using the Sign-In Widget and your Okta cookie isn't set, try enabling third-party
```
Comment (rchild-okta) [inline _source/_code/javascript/okta_sign-in_widget.md]: Is this line break intentional?

Comment (mystiberry-okta) [inline _source/_code/javascript/okta_sign-in_widget.md]: it builds properly. Still breaking my habit of ignoring line breaks as irrelevant.

**Pull request merged**